src/OnchainKitProvider.tsx

```diff
@@ -20,9 +22,15 @@ export function OnchainKitProvider({
   const value = useMemo(() => {
     return {
       address: address ?? null,
+      apiKey: apiKey ?? null,
+      rpcUrl:
+        rpcUrl ??
+        (apiKey
+          ? `https://api.developer.coinbase.com/rpc/v1/${chain.name.replace(' ', '-').toLowerCase()}/${apiKey}`
```

Comment:
We should us the new `getRpcUrl` here.

Reply:
does the context also make a call to `setOnchainKitConfig`?

every method i am implementing depends on the `ONCHAIN_KIT_CONFIG` object, I'm not sure if this configuration is passed by the provider/context

Reply:
basically if I use `getRpcUrl` here it will error out because the `ONCHAIN_KIT_CONFIG` object hasn't been set yet